examples/cow_sphere.rs

```diff
@@ -142,3 +149,16 @@ fn fibonacci_spiral(n: usize) -> impl Iterator<Item = DVec3> {
 fn lerp(a: f64, b: f64, t: f64) -> f64 {
     a * (1.0 - t) + b * t
 }
+
+// Send an actionbar message to all clients when their flying state changes.
+fn display_is_flying(
+    mut clients: Query<(&mut Client, &PlayerAbilitiesFlags), Changed<PlayerAbilitiesFlags>>,
+) {
+    for (mut client, abilities_flags) in clients.iter_mut() {
+        if abilities_flags.flying() {
+            client.send_action_bar_message("You are flying!".into_text().color(Color::GREEN));
+        } else {
+            client.send_action_bar_message("You are not flying!".into_text().color(Color::RED));
+        }
+    }
+}
```

Comment:
At this point I think it would be best to create a dedicated example for abilities.

Reply:
I don't really know because in reality, `cow_sphere` wasn't showing much and I think `Abilities` are not this important, but they have demonstration in some example so that if someone ask how to use it we can just redirect them.

Reply:
That's fine. In the future I would like the examples to be more complex so we don't have to keep making examples for every tiny feature we add.